internal/pkg/describe/app.go

```diff
@@ -55,6 +56,7 @@ func (a *App) HumanString() string {
 	}
 	fmt.Fprintf(writer, "  %s\t%s %s\n", "Version", a.Version, availableVersion)
 	fmt.Fprintf(writer, "  %s\t%s\n", "URI", a.URI)
+	fmt.Fprintf(writer, "  %s\t%s\n", "Permissions Boundary", a.PermissionsBoundary)
```

Comment:
nit: I wonder if we should write "N/A" if there is no setting for these fields what do you think?
So that it looks like this when empty:
```
About

  Name                  demo
  Version               v1.1.0 (latest available: v1.0.2)
  URI                   N/A
  Permissions Boundary  N/A
```